### DISCUSSION TITLE
Projects Folder Deleted Overnight

### DISCUSSION THREAD

**Discussion by MichaelThomas0721** [coder/code-server#6268] [Q&A] [answered]
I am not sure if this is a bug or a mistake on my end but my entire projects folder was deleted overnight. I had almost everything backed up so not a big deal, I just don't want to make the same mistake.

I created the directory `/projects` and that is where I have been keeping all my projects but it was deleted, I have a feeling it is because I created it outside of the workspaces folder but I don't want to make any assumptions. It had been a few weeks since I initially created the folder which is why this surprised me.

I am using the docker version of code-server.

If someone could confirm my suspension that would be awesome.

Thanks

**Comment by code-asher** [comment, marked as answer]
Were you mounting `/projects` to the host?  If not then it would be ephemeral and lost when the container restarts.

If that directory was mounted or if the container has not restarted then that I am not sure what could have caused it.

**Reply by MichaelThomas0721** [reply]
I didn't mount it, I created it within code-server using the command line. Thanks for clearing it up!
